pulp_smash/pulp3/pytest_plugin/__init__.py

```diff
@@ -0,0 +1,32 @@
+import asyncio
+
+import pytest
+
+from pulp_smash.config import get_config
+
+from pulpcore.client.pulpcore import ApiClient, TasksApi
+
+
+@pytest.fixture
+def pulpcore_client():
+    cfg = get_config()
+    configuration = cfg.get_bindings_config()
+    return ApiClient(configuration)
+
+
+@pytest.fixture
+def tasks_api_client(pulpcore_client):
+    return TasksApi(pulpcore_client)
```

Comment:
I pretty much like this. Can we mark it as scope="session"?

Reply:
I think this was a very good idea.